src/components/split-button/split-button.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, boolean, select } from '@storybook/addon-knobs';
+import SplitButton from './split-button';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import Button from '../button';
+import notes from './notes.md';
+
+storiesOf('Split Button', module).add(
+  'default',
+  () => {
+    const as = select('as', OptionsHelper.themesBinary, SplitButton.defaultProps.as);
+    const dataElement = text('data-element');
+    const dataRole = text('data-role');
+    const disabled = boolean('disabled', SplitButton.defaultProps.disabled);
+    const textContent = text('text', 'Example Split Button');
+
+    return (
+      <SplitButton
+        as={ as } data-element={ dataElement }
+        data-role={ dataRole } disabled={ disabled }
+        text={ textContent }
+      >
+        <Button>Example Button</Button>
+        <Button>Example Button</Button>
+        <Button>Example Button</Button>
+      </SplitButton>
+    );
+  },
+  { notes: { markdown: notes } }
+);
```

Comment:
Unneeded Button props being shown:

![Screenshot 2019-03-13 at 15 14 00](https://user-images.githubusercontent.com/4471057/54290330-a8c4d080-45a2-11e9-8605-227a7eb7ee5f.png)

You can get rid of specific tables using the propTablesExclude property on the info addOn. Check out some of the existing Storybook work for examples, but effectively:

```
  .addParameters({
    info: {
      propTablesExclude: [Button, State]
    }
  })
```